convert_changelog.py

```diff
@@ -0,0 +1,9 @@
+import re
+
+
+with open("docs/source/changelog.rst") as f:
+    text = f.read()
+
+# :issue:`312` -> https://github.com/googleapis/python-bigquery-pandas/issues/312
+c = re.compile(r":issue:`([0-9]+)`", flags=re.MULTILINE)
+print(re.sub(c, r"[#\1](https://github.com/googleapis/python-bigquery-pandas/issues/\1)", text))
```

Comment:
Why is this checked in? Isn't it a one-time script?

Reply:
Oops, good catch. Yeah, just a one-time script. Not even the final version I used (which is in the PR description)